metrics/src/main/java/ai/vespa/metrics/set/MicrometerMetrics.java

```diff
@@ -0,0 +1,64 @@
+package ai.vespa.metrics.set;
+
+import ai.vespa.metrics.Unit;
+import ai.vespa.metrics.VespaMetrics;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.EnumSet;
+
+import static ai.vespa.metrics.Suffix.count;
+import static ai.vespa.metrics.Suffix.last;
+import static ai.vespa.metrics.Suffix.max;
+import static ai.vespa.metrics.Suffix.sum;
+
+public enum MicrometerMetrics implements VespaMetrics {
+    JVM_MEMORY_COMMITTED("jvm.memory.committed",Unit.BYTE, "The amount of memory in bytes that is committed for the Java virtual machine to use"),
+    JVM_MEMORY_MAX("jvm.memory.max", Unit.BYTE, "The maximum amount of memory in bytes that can be used for memory management"),
+    JVM_MEMORY_USED("jvm.memory.used",Unit.BYTE, "The amount of used memory"),
+    JVM_BUFFER_COUNT("jvm.buffer.count",Unit.BUFFER, "An estimate of the number of buffers in the pool"),
+    JVM_BUFFER_MEMORY_USED("jvm.buffer.memory.used",Unit.BYTE, "An estimate of the memory that the Java virtual machine is using for this buffer pool"),
+    JVM_BUFFER_TOTAL_CAPACITY("jvm.buffer.total.capacity",Unit.BYTE, "An estimate of the total capacity of the buffers in this pool"),
+    JVM_CLASSES_LOADED("jvm.classes.loaded",Unit.CLASS, "The number of classes that are currently loaded in the Java virtual machine"),
+    JVM_CLASSES_UNLOADED("jvm.classes.unloaded",Unit.CLASS, "The total number of classes unloaded since the Java virtual machine has started execution"),
+    JVM_GC_PAUSE("jvm.gc.pause",Unit.MILLISECOND, "Time spent in GC pause"),
```

Comment:
Timer metrics are reported in seconds

Reply:
Fixed with [64b95c0](https://github.com/vespa-engine/vespa/pull/33189/commits/64b95c08d2b71c402bd427dc59f149a0737d53f4)